packages/components/src/components/linear-progress/linear-progress.tsx

```diff
@@ -1,7 +1,13 @@
 import classNames from 'classnames';
 import React from 'react';
 
-export const LinearProgress = ({ progress }) => (
+type TLinearProgress = {
+    progress: number;
+    className: string;
+    height: number;
+};
+
+export const LinearProgress = ({ progress }: Partial<TLinearProgress>) => (
```

Comment:
Seems only progress param is used in the component. If so, can we remove this Partial typing?